README.md

```diff
@@ -4,10 +4,11 @@ This is the official repository for all things regarding the ForkDela smart cont
 All contracts are located in the `contracts` folder.
 
 ## Summary
+- [Development](#development)
 - [Libraries](#libraries)
-  - [SafeMath](#library-safemath)
```

Comment:
All of the link hashes here need to be changed if you change the headers down below

Reply:
Fixed with the next commit.